vsc_pdf_template_transformer/lib/utils/evaluator.dart

```diff
@@ -88,3 +89,84 @@ List<T>? evaluateList<T>(dynamic expression, Map<String, dynamic> data) {
 
   return List<T>.from(result, growable: false);
 }
+
+T? evaluateEnum<T extends Enum>(
```

Comment:
I added this method to make conversion of enums easier, less code, and less err prone. I adapted the other enum-based methods to use it.